modules/tracking/test/test_trackers.cpp

```diff
@@ -534,6 +547,13 @@ TEST_P(DistanceAndOverlap, DISABLED_GOTURN)
   test.run();
 }
 
+TEST_P(DistanceAndOverlap, Scaled_Data_MOSSE)
+{
+  TrackerTest test( TrackerMOSSE::create(), dataset, 22, -0.05f, Scale_1_1, 1);
```

Comment:
Why do you have a negative value here? If the tracker failed, 0 is enough. I have good results on Ubuntu 16.04 in this test (min overlap is 0.69).

Reply:
oh, i simply must have misread your comment about the values above.

i'll try again ;)